### ISSUE TITLE
Simplify headers associated with sources files

### ISSUE THREAD

**Issue by jcfr** [Slicer/Slicer#6281]
## Is your feature request related to a problem? Please describe.

The information included in headers associated with c++, python or CMake source files is not consistently organized (see examples below).

There is a lack of guidelines for integrating grant information and referencing contributor in case of derived work.

## Open Questions

* When using module & extension templates, should the header be updated or removed ?

## Describe the solution you'd like

Few possible approaches:

### Simple one-liner

This is the approach implemented by the CMake project where we now have the folloing:

```
# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
# file Copyright.txt or https://cmake.org/licensing for details.
```

References:
* https://github.com/Kitware/CMake/commit/86578eccf2e82286248796bad1032cd0e3a5e1e2
* https://github.com/Kitware/CMake/blob/master/Copyright.txt
* https://cmake.org/licensing

## Describe alternatives you've considered

Have a more verbose header automatically updated. It would be between `auto` comment. 

That said, for sake of consistency, we would need to have this support for CMake, C++ and python files.

## Additional context

NA

#### Original contributor is not mentioned

https://github.com/Slicer/Slicer/blob/a258951f473d74f24a93cff95673ea6163a7b5e1/Modules/Loadable/Markups/qSlicerMarkupsModuleWidget.h#L1-L16

#### Original contributor is mentioned along side with grant number

https://github.com/Slicer/Slicer/blob/a258951f473d74f24a93cff95673ea6163a7b5e1/Libs/MRML/DisplayableManager/vtkMRMLAbstractDisplayableManager.h#L1-L19

#### There is no header information included 

https://github.com/Slicer/Slicer/blob/a258951f473d74f24a93cff95673ea6163a7b5e1/Modules/Loadable/Volumes/Widgets/qSlicerScalarVolumeDisplayWidget.h#L1-L8

**Comment by lassoan** [comment]
A one-liner like this could work:

```
Distributed under 3D Slicer license. See accompanying file LICENSE.txt or https://www.slicer.org/license for details.
```

In addition to this, we should preserve additional information about funding grants.

I find listing individual contributor names redundant (already recorded in revision control) and misleading (most likely many more people contributed over time). We could ask what people think about removing these names (maybe on the Slicer forum?). I would certainly not mind if my name was removed from files, if I put my name in any file then I only did it to follow the existing pattern.

**Comment by pieper** [comment]
Yes, I agree about including names in the source code is misleading (e.g. Bill, Will, and Ken's name in all the VTK files is just a historical artifact and it always seems a bit odd).  The git history and project week pages are better ways of acknowledging contributions.

**Comment by jamesobutler** [comment]
Changes to acknowledging contributors would also be for scripted modules as well? Thinking about a commonly used module like Segment Editor which had some core contributors at original introduction, but has the likely chance of being contributed by many people over time. Git history provides a complete history of contributors for the module.
https://github.com/Slicer/Slicer/blob/fc64d915f9e8e7401bd85cc9d7800b58464ca283/Modules/Scripted/SegmentEditor/SegmentEditor.py#L9-L25
